app/client/rpc/utils.go

```diff
@@ -0,0 +1,27 @@
+package rpc
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+func PopModel(_ http.ResponseWriter, r *http.Request, _ httprouter.Params, model interface{}) error {
```

Comment:
I'm guessing `Model` is a common semantic name for something in the routing world but not familiar with it. Could you shed more details for my personal learning?